cpp/include/cugraph/prims/transform_reduce_e.cuh

```diff
@@ -117,9 +121,7 @@ __global__ void for_all_major_for_all_nbr_hypersparse(
     idx += gridDim.x * blockDim.x;
   }
 
-  e_op_result_sum =
-    block_reduce_edge_op_result<e_op_result_t, transform_reduce_e_for_all_block_size>().compute(
-      e_op_result_sum);
+  e_op_result_sum = BlockReduce(temp_storage).Reduce(e_op_result_sum, edge_property_add);
```

Comment:
As we discussed, if block_reduce_edge_op_result can serve what we need, we'd better delete this and directly use cub everywhere for consistency.

Reply:
`block_reduce_edge_op_result` is still being used by `copy_v_transform_reduce_nbr` and therefore cannot be removed yet. It should be removed though because it is buggy. Perhaps in another PR?

Reply:
Yeah... you can remove it when you write test cases for copy_v_transform_reduce_nbr.